subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmDependencies.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.plugins.jvm;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.artifacts.ProjectDependency;
+import org.gradle.api.artifacts.dsl.Dependencies;
+import org.gradle.api.attributes.CompileView;
+import org.gradle.api.model.ObjectFactory;
+
+import javax.inject.Inject;
+
+/**
+ * APIs that are available for {@code dependencies} blocks used by JVM components.
+ *
+ * @apiNote This interface is intended to be used to mix-in methods that expose JVM-ecosystem specific dependencies to the DSL.
+ * @implSpec The default implementation of all methods should not be overridden.
+ *
+ * @since 8.0
+ */
+@Incubating
+public interface JvmDependencies extends Dependencies {
```

Comment:
🧐 I think the name is too general.  Maybe `JvmInternalProjectDependencies` would be better?